torch/csrc/jit/frontend/concrete_module_type.h

```diff
@@ -83,6 +86,8 @@ class VISIBILITY_HIDDEN ConcreteModuleTypeBuilder {
   void addFailedAttribute(std::string name, std::string failureReason);
   void addIgnoredAttribute(std::string name);
   void setIterableModuleKind(IterableModuleKind kind);
+  // Set a type hint on this Module (perhaps an InterfaceType).
```

Comment:
mind clarifying the comment ? does this apply to the module itself or its contained modules

Reply:
Yep, I will include the explanations from these comment threads in the code itself.